templates/comments/media-js.html

```diff
@@ -217,6 +217,16 @@
                 $comment.find('.comment-body').show();
                 $comment.find('.bad-comment-body').hide();
             };
+
+            var code_regex = [/input\(/, /#include/, /void\s+main/, /fn\s+main/, /func\s+main/];
+            $(document).on('click', 'form.comment-submit-form .button[type=submit]', function (e) {
+                var text = $(this).parents('form').find('#id_body').val();
+                if (code_regex.some(function (regex) {return regex.test(text);})) {
+                    if (!confirm({{ _('Looks like you\'re trying to post a source code!\n\nThe comment section is not for posting source code.\nIf you want to submit your solution, please use the "Submit solution" button.\n\nAre you sure you want to post this?')|htmltojs }})) {
```

Comment:
The English here feels a little off. Could we rewrite it? Specifically "post a source code".

Reply:
done. the message is changed to:

```
Looks like you're trying to post some source code!

The comment section is not for posting source code.
If you want to submit your solution, please use the "Submit solution" button.

Are you sure you want to post this?
```